### DISCUSSION TITLE
Is there a way to hide hidden users?

### DISCUSSION THREAD

**Discussion by johnsoep** [pnp/sp-dev-fx-webparts#4821] [Q&A]
This is for the react-directory webpart.

I'm looking to deploy this webpart for my organization but we utilize hidden accounts to protect users who have applied for confidential status and we don't want them to be displayed in the directory. I've hidden several accounts in my Dev environment and the directory webpart is still showing them. Is there a way that I can modify the search function to force where the hidden attribute is false?

**Comment by netmatrixtech** [comment]
Please see that I've created my own enhancements to react-directory to facilitate filtering of results based on Entra ID Group Object ID. It'll be up to the contributors for this webpart to decide if they wish to add it into their project.
It should cater to most of your requests if you use an Entra Dynamic Group to apply the accounts you want shown in the results of the webpart.

[https://github.com/pnp/sp-dev-fx-webparts/discussions/6463](https://github.com/pnp/sp-dev-fx-webparts/discussions/url)
cheers
Tim

**Reply by johnsoep** [reply]
Awesome! Thanks I'll definitely give it a look!
